lib/email_checker/smtp.ex

```diff
@@ -26,11 +30,19 @@ defmodule EmailChecker.SMTP do
     |> EmailChecker.MX.lookup
   end
 
+  defp timeout_opt() do
```

Comment:
Same question about empty parentheses and same or new line for case return values